src/EnergyPlus/DXCoils.cc

```diff
@@ -7549,22 +7487,19 @@ void SizeDXCoil(EnergyPlusData &state, int const DXCoilNum)
             PrintFlag = true;
             state.dataSize->DataTotCapCurveIndex = thisDXCoil.CCapFTemp(Mode);
             if (thisDXCoil.DXCoilType_Num == HVAC::CoilDX_CoolingTwoStageWHumControl) {
-                SizingMethod = HVAC::CoolingCapacitySizing;
```

Comment:
@Nigusse @rraustad Are these changes to DXCoils.cc supposed to be in this PR?

Reply:
It looks like cleaning up unused information. I guess this can be done here as long as diffs show only for HSPF calculations.

Reply:
> @Nigusse @rraustad Are these changes to DXCoils.cc supposed to be in this PR?

I think, @Myoldmopar might have made sizing related cleanup.

Reply:
Ah, ok, I see that commit.